includes/wizards/class-seo-wizard.php

```diff
@@ -70,7 +70,122 @@ public function get_length() {
+				'permission_callback' => [ $this, 'api_permissions_check' ],
+			]
+		);
+		register_rest_route(
+			'newspack/v1/wizard/' . $this->slug,
+			'settings',
+			[
+				'methods'             => \WP_REST_Server::EDITABLE,
+				'callback'            => [ $this, 'api_update_seo_settings' ],
+				'permission_callback' => [ $this, 'api_permissions_check' ],
+				'args'                => [
+					'under_construction' => [
+						'sanitize_callback' => 'absint',
+					],
+					'separator'          => [
+						'sanitize_callback' => 'sanitize_text_field',
+					],
+					'verification'       => [],
+					'urls'               => [],
+				],
+			]
+		);
+	}
+
+	/**
+	 * API endpoint to retrieve all settings necessary to render the SEO wizard.
+	 *
+	 * @return WP_REST_Response with the info.
+	 */
+	public function api_get_seo_settings() {
+		$response = $this->get_seo_settings();
+		return rest_ensure_response( $response );
+	}
+
+	/**
+	 * Update SEO wizard settings.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response with the info.
+	 */
+	public function api_update_seo_settings( $request ) {
+		$cm = Configuration_Managers::configuration_manager_class_for_plugin_slug( 'wordpress_seo' );
+		if ( isset( $request['urls'] ) ) {
+			$urls = $request['urls'];
+			if ( isset( $urls['facebook'] ) ) {
+				$cm->set_option( 'facebook_site', $urls['facebook'] );
+			}
+			if ( isset( $urls['twitter'] ) ) {
+				$cm->set_option( 'twitter_site', $urls['twitter'] );
+			}
+			if ( isset( $urls['instagram'] ) ) {
+				$cm->set_option( 'instagram_url', $urls['instagram'] );
+			}
+			if ( isset( $urls['linkedin'] ) ) {
+				$cm->set_option( 'linkedin_url', $urls['linkedin'] );
+			}
+			if ( isset( $urls['youtube'] ) ) {
+				$cm->set_option( 'youtube_url', $urls['youtube'] );
+			}
+			if ( isset( $urls['pinterest'] ) ) {
+				$cm->set_option( 'pinterest_url', $urls['pinterest'] );
+			}
+		}
+		if ( isset( $request['verification'] ) ) {
+			$verification = $request['verification'];
+			if ( isset( $verification['bing'] ) ) {
+				$cm->set_option( 'msverify', $verification['bing'] );
+			}
+			if ( isset( $verification['google'] ) ) {
+				$cm->set_option( 'googleverify', $verification['google'] );
+			}
+		}
+		if ( isset( $request['title_separator'] ) ) {
+			$cm->set_option( 'separator', $request['title_separator'] );
+		}
+		if ( isset( $request['under_construction'] ) ) {
+			$environment_type = absint( $request['under_construction'] ) ? 'staging' : 'production';
+			$cm->set_option( 'environment_type', $environment_type );
+		}
+		$response = $this->get_seo_settings();
+		return rest_ensure_response( $response );
+	}
+
+	/**
+	 * Retrieve all settings necessary to render the SEO wizard.
+	 *
+	 * @return Array with the info.
+	 */
+	public function get_seo_settings() {
+		$cm = Configuration_Managers::configuration_manager_class_for_plugin_slug( 'wordpress_seo' );
+
```

Comment:
Like above, a quick `is_configured` check to make sure Yoast is active will prevent fatal errors.

Reply:
See: https://github.com/Automattic/newspack-plugin/pull/435#discussion_r379922953